0035-ilp-over-http/0035-ilp-over-http.md

```diff
@@ -52,7 +59,78 @@ Content-Type: application/octet-stream
 < Body: Binary OER-Encoded ILP Fulfill or Reject Packet >
 ```
 
-All ILP Packets MUST be returned with the HTTP status code `200: OK`.
+If the request included a `Request-Id` header, the recipient handling the ILP Prepare SHOULD choose to handle the packet asynchronously and return the corresponding ILP Fulfill/Reject in a separate outgoing HTTP request.
```

Comment:
Thanks @kincaidoneil for pushing the PR forward - overall I like the proposal.

Are we sure we want to vary the behavior of the server based upon the presence of the `Request-Id` header? I'm not aware of any standard around doing such a thing, but this RFC seems like a good start: https://tools.ietf.org/html/rfc7240#section-4.1

That said, some alternatives for varying the response would be:

* Use a different URL prefix (e.g., `/ilp` vs `/ilp-async`)
* Use a different Media Type (`application/ilpv4-async` or `application/ilp-over-http-async`) or something like that.

The primary reason I bring this up is that it might be useful for an ILP-over-HTTP sync request/response that wants to use idempotence to _also_ use both the requestId and the `Idempotence Key` without triggering async behavior, (e.g., for logging or audit purposes).

Reply:
I'd be in favor of `Prefer: respond-async`